File: 0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {

        for (int i = 0; i < nums.length; i ++)
        {
            for (int x = i + 1; x < nums.length; x ++)
            {
                if (nums[i] + nums[x] == target)
                {
                    int[] twosum = {i, x};
                    return twosum;
                }
            }
        }
        return new int[2];
    }
}

File: 0001-two-sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        h = dict()
        for ind, val in enumerate(nums):
            h[val] = ind
        for idx, v in enumerate(nums):
            if target - v in h and h[target - v]!=idx:
                return [h[target - v], idx]
    
        
